src/scss/constants.scss

```diff
@@ -79,6 +79,8 @@ $announcement-container-width-admin: 600px;
 $announcement-container-width: 470px;
 $announcement-container-height: 140px;
 $announcement-banner-admin-height: 55px;
+$video-participant-optimal-width: 260px;
+$video-participant-optimal-height: 145px;
```

Comment:
[NOTE]
If only these were relative or clamped relative values, it'd make it easy for too small or too big screen size. Good enough for now.